### DISCUSSION TITLE
[RRFC] package maintainers available for hire

### DISCUSSION THREAD

**Discussion by brody2consult** [npm/rfcs#61] [General]
There are package maintainers such as myself who are available for hire and may be willing to offer some exclusive support services. Some looking for part-time work, some looking for short-term contracts, and others looking for long-term employment. Some may be remote only and others may be available in a major city. I think this is something very different from funding with a "fund" command (npm/cli#273)  which relies on the donation model.

As suggested by @darcyclarke in <https://github.com/npm/cli/pull/273#issuecomment-548193740>, this could be done based off the [`PACKAGE-SUPPORT` schema](https://github.com/nodejs/package-maintenance/blob/master/docs/drafts/PACKAGE-SUPPORT.md) from the [`nodejs/package-maintenance` working group](https://github.com/nodejs/package-maintenance/). From a quick look, it does not seem to cover all the cases I mentioned and likely some other cases I have not even thought of.

I would be happy to work on a RFC for this but really do not know where I should start. Should I raise an RFC on npm (this repository), discuss this in  [`nodejs/package-maintenance` working group](https://github.com/nodejs/package-maintenance/), or do something else? Any pointers to what kind of information and what kind of detail should be included?

**Comment by wesleytodd** [comment]
Hey @brodybits, it would be great if you have actionable suggestions on how the support schema can cover your use cases and then open issues on the WG.

**Comment by brody2consult** [comment]
Thanks @wesleytodd for the response. First, I just added [RRFC] to the title, should have added it before.

In terms of "actionable suggestions", as I said in the description, my problem is that I really do not know where to start. Any pointers would be very highly appreciated.

**Comment by wesleytodd** [comment]
A PR or an issue with update reccomendations to the document we have in draft would be good.  Here are some links:

The draft: https://github.com/nodejs/package-maintenance/blob/master/docs/drafts/PACKAGE-SUPPORT.md
An example change I made: https://github.com/nodejs/package-maintenance/pull/254
The long original PR: https://github.com/nodejs/package-maintenance/pull/220
